test/pummel/test-heapdump-zlib.js

```diff
@@ -3,15 +3,27 @@
 require('../common');
 const { validateSnapshotNodes } = require('../common/heap');
 const zlib = require('zlib');
+const { mustCall } = require('../common');
 
 validateSnapshotNodes('Node / ZlibStream', []);
 // eslint-disable-next-line no-unused-vars
```

Comment:
This can be removed now since `gzip` is used.

Reply:
Thanks for noticing this. Fixed, as well as the `common` import.